modules/dicom_archive/php/dicom_archive.class.inc

```diff
@@ -71,12 +71,8 @@ class Dicom_Archive extends \DataFrameworkMenu
     function getFieldOptions() : array
     {
         // get the list of sites available for the user
-        $list_of_sites = \Utility::getSiteList();
-        $site_options  = array();
-
-        foreach ($list_of_sites as $id => $name) {
-            $site_options[$name] = $name;
-        }
+        $site_names   = array_values(\Utility::getSiteList());
+        $site_options = array_combine($site_names, $site_names);
```

Comment:
this works, i just find it more confusing. I think removing the `$id` would have had the same effect.

not a breaking change though

Reply:
I'll change the comment to make it more clear